### PR TITLE
Add wordpress

### PR DESCRIPTION
**Pull request by marians**
RFR @puja108

### CODE REVIEW

File: wordpress/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+We use the version `5.5` of the official MySQL component, mainly because it has a lean default configuration that doesn't use a lot of memory. More information is available at
+
+https://hub.docker.com/r/library/mysql/
+
+## About the `swarm.json` file
+
+* The definition contains the two components mentioned before on the top level.
+* The `wordpress` component links to the `mysql` component to allow the database connection from WordPress's PHP code to MySQL.
+* Both components have a volume to store data throughout restarts.
```
Comment (puja108) [inline wordpress/README.md]: We don't use volumes in docker-compose, which is intentional I presume?

**Comment by puja108** [comment]
tiny comment, other than that LGTM

File: wordpress/README.md (hunk 1 above)
Comment (marians) [inline wordpress/README.md]: Yes, this is intentional, since it's another source of error. Also in the docker run commands.

**Comment by marians** [comment]
Thanks!

**Pull request merged**